src/test/java/bio/terra/service/resourcemanagement/ProfileConnectedTest.java

```diff
@@ -0,0 +1,170 @@
+package bio.terra.service.resourcemanagement;
+
+
+import bio.terra.app.configuration.ConnectedTestConfiguration;
+import bio.terra.common.category.Connected;
+import bio.terra.common.fixtures.ConnectedOperations;
+import bio.terra.common.fixtures.ProfileFixtures;
+import bio.terra.model.CloudPlatform;
+import bio.terra.service.configuration.ConfigurationService;
+import bio.terra.service.iam.IamProviderInterface;
+import bio.terra.service.profile.ProfileDao;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.iterableWithSize;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class ProfileConnectedTest {
```

Comment:
I feel like I'm missing something...why is this a connected test and not a unit test?  Does this actually hit cloud resources?